utils.h

```diff
@@ -71,6 +71,15 @@ int posix_fadvise(int fd, off_t offset, off_t len, int advice);
  */
 #define posix_fadvise(fd, offset, len, advice) (0)
 
+/*
+ * If included after
+ * #define _POSIX_C_SOURCE 200112L
+ * #define _XOPEN_SOURCE 600
+ * unistd.h seems not to declare these:
+ */
+int pledge(const char*, const char*);
+int unveil(const char*, const char*);
+
```

Comment:
Doesn't OpenBSD have a header for these system calls? Move them to the preprocessor block around `restrict_fsaccess()` in `utils.c`.

Reply:
It’s unistd.h, I think I'll just move `#include <unistd.h>` above _POSIX_C_SOURCE/_XOPEN_SOURCE where necessary...